tests/foreman/ui/test_hostunification.py

```diff
@@ -317,6 +325,50 @@ def test_positive_unregister_content_host(self):
                 self.assertIsNotNone(self.contenthost.search(vm.hostname))
                 self.assertIsNotNone(self.hosts.search(vm.hostname))
 
+    @run_in_one_thread
+    @run_only_on('sat')
+    @tier3
+    def test_positive_delete_content_host(self):
+        """Unregister and delete a host from content-hosts page
+
+        @id: 3c75c1e6-85e3-49e6-a10e-6052a0db2b7f
+
+        @steps:
+        1.  Un-register and delete a host from content-host page
+        2.  View host under content hosts
+        3.  View host under 'All hosts'
+
+        @assert: Hosts un-registered from content-host should disappear from
+        both sides of UI
+
+        @CaseLevel: System
+        """
+        org = entities.Organization().create()
```

Comment:
That's another module which doesn't use `session_org` approach so .create isn't called twice. In that file half of the tests are 'fakes' so nothing serious is created and that's why they share the same org, another half works with real hosts so just in case they use separate org per test. Using `session_org` approach will create separate org for each test, which is redundant.